### ISSUE TITLE
Bug: Experiment 1 summary incorrectly states training steps

### ISSUE THREAD

**Issue by bg-l2norm** [vukrosic/analysis-of-muon-optimizer-in-llms#1]
There is a discrepancy between the documented training steps for Experiment 1 and the actual value in the code.
- the markdown summary (**and paper!**) states 1000 steps.
- the code (exp1_baseline.py) sets max_steps to 5000.

<img width="890" height="141" alt="Image" src="https://github.com/user-attachments/assets/bcf9c036-fc67-477b-a3d3-02313cdf16d4" />

I couldn't understand why Experiment 1's validation loss (0.0476) was so much lower than Experiment 2's (2.5347). Turns out the former was trained for 5000 steps. Specifying this in the documentation would help avoid such confusion. :)
